modules/flowable-engine-common/src/main/java/org/flowable/common/engine/impl/db/AbstractDataManager.java

```diff
@@ -19,7 +19,6 @@
 import java.util.List;
 import java.util.Map;
 import java.util.function.Consumer;
-
```

Comment:
This is a very minor issue that has no effect on the execution of the code but as per the other PR where I pointed you to the IDE settings there is a blank line between groups of imports that you are removing here.

Reply:
Indeed, trying to manually keep these correct is difficult.

As there was no .importorder configuration for Eclipse available, I created one that matches the IntelliJ style and appears to provide the correct results.